### PR TITLE
adding things to the kill command

### LINKED ISSUE #581: adding documentation for kill

### PR DESCRIPTION
**Pull request by lgdgodv**
# Advanced HW 5 submission

Closes #581 

Name: Muting Wu
Uniqname: wumuting

### CODE REVIEW

File: _commands/basics/kill.md [hunk 1]
```diff
@@ -3,12 +3,13 @@
...
 --
-TODO: Add documentation for this command by submitting a pull request.
+The kill command is used on Linux systems to terminate process by its process id without having to reboot the system.
 <!-- one line explanation would go here -->
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+The syntax is kill [signal or option] PID(s)
+$kill 27410         Means killing a current running process with process id 27410
```
Comment (cyanliu) [inline _commands/basics/kill.md]: Could you use a comment or some other change of format so it doesn't seem like `Means killing a current running process with process id 27410` is a part of the command?

File: _commands/basics/kill.md [hunk 2]
```diff
@@ -17,8 +18,32 @@ TODO: Add documentation for this command by submitting a pull request.
 
 #### Example command
 
-##### Break it down
+##### kill -9 PID
+Actually, kill is a command to send signals to the specified process, each process has a set of standard handlers by the operating systems, and signal 9 called SIGKILL can almost guarantee to kill a process if the default signal 15 can't kill it.
```
Comment (cyanliu) [inline _commands/basics/kill.md]: With any instance of the command, it should be styled as `inline code` using backticks (`). So the beginning of your documentation should look something like

>Actually, `kill` is a command...

File: _commands/basics/kill.md [hunk 3]
```diff
@@ -17,8 +18,32 @@ TODO: Add documentation for this command by submitting a pull request.
...
 
 #### Example command
 
-##### Break it down
+##### kill -l 
+Although kill -9 PID is the most common kill commands and you probably don't need to know more, but if you really need to know the full list of about 60 different signals, type kill -l
+
+#### Example command
+
+##### ps -aux | less
```
Comment (cyanliu) [inline _commands/basics/kill.md]: could you show example output of this command, and what value would correspond to the process ID?

File: _commands/basics/kill.md [hunk 4]
```diff
@@ -17,8 +18,32 @@ TODO: Add documentation for this command by submitting a pull request.
...
 
-##### Break it down
+##### kill -l 
+Although kill -9 PID is the most common kill commands and you probably don't need to know more, but if you really need to know the full list of about 60 different signals, type kill -l
+
+#### Example command
+
+##### ps -aux | less 
+Yes, you want to kill a process, but you first need to know the process id of that process. By the command above you can see brief information of current running processes including its PID, thus you can kill that process by its process id
+~
```
Comment (cyanliu) [inline _commands/basics/kill.md]: everything from this point onward should be removed, all of the `~` is unnecessary

**Comment by tarunsk** [comment]
You have not been awarded credit for this PR, as none of the requested changes were made even though you were provided sufficient time to do so.

**Comment by lgdgodv** [comment]
This shouldn't be the case.
I made changes 23 days ago, you could click this link to see
https://github.com/lgdgodv/c4cs.github.io/blob/master/_commands/basics/kill.md


On Wed, Apr 4, 2018 at 3:25 PM, Tarun Khubchandani <notifications@github.com

**Comment by lgdgodv** [comment]
And I am pretty sure I change all the required changes.

**Comment by tarunsk** [comment]
I think I see the issue. You made the changes on a different branch in your forked repository, so it is not reflecting in this PR. If you can merge those changes into this branch, we can re-evaluate this PR.

**Comment by lgdgodv** [comment]
I just made the same change to my old branch, is that correct right now?
Thanks

**Comment by lgdgodv** [comment]
I made changes yesterday, is that ok?

On Thu, Apr 5, 2018 at 4:00 AM, Tarun Khubchandani <notifications@github.com

**Comment by tarunsk** [comment]
You have been awarded credit for advanced homework 5! We will not be merging this in for the time being as we are in the process of archiving the site, but you have still been awarded credit. Your grade on Canvas will not show this change immediately, but it will reflect the next time we update all grades on Canvas. Congratulations!

**Pull request closed without merging**